presto-kinesis/src/test/java/io/prestosql/plugin/kinesis/TestMinimalFunctionality.java

```diff
@@ -100,7 +100,7 @@ public void stop()
     public void spinUp(String accessKey, String secretKey)
```

Comment:
There is `presto-kinesis/src/main/java/io/prestosql/plugin/kinesis/KinesisConnectorFactory.java` that is using `System.setProperty`

Reply:
Yep. There is a TODO there which needs to be addressed.